docs/definitions/using.md

```diff
@@ -1,6 +1,105 @@
-# using the definitions programatically
+# Using the definitions programatically
 
-required reading: [definitions overview](definitions) / [fetching definitions](/definitions/fetching)
+**Required reading** 
+* [fetching stuff](/api)
+* [definitions overview](/definitions)
+* [fetching definitions](/definitions/fetching)
+* [fetching inventories](/inventory)
+* [itemType API reference](https://bungie-net.github.io/multi/schema_Destiny-DestinyItemType.html#schema_Destiny-DestinyItemType).
 
-## what goes here?
-what to do once you have the defs in memory
+Once the definitions have been fetched, you should decide if you want to use the JSON or SQLite format.
+
+In this example we will use the response to get the [DestinyInventoryItemDefinition](https://bungie-net.github.io/multi/schema_Destiny-Definitions-DestinyInventoryItemDefinition.html#schema_Destiny-Definitions-DestinyInventoryItemDefinition) object. The response also separates all the definition objects via language, for obvious reasons, which means we have to use the corresponding country code to get the correct object. In this scenario the country code will be `en` (english).
+
+**JSON**
+
+> `Response.jsonWorldComponentContentPaths.en.DestinyInventoryItemDefinition`
+
+**SQLite**
+
+> `query goes here`
+
+
+To make the process easier for yourself later on, consider declaring a variable that you assign the definitions object to; using whichever route you went to fetch the definitions (SQLite or JSON).
+
+```js
+const inventoryItemDefinitions = Response.jsonWorldComponentContentPaths.en.DestinyInventoryItemDefinition;
```

Comment:
this seems... wrong? this leads to a single string that's a URL. not to the definitions info as line 29 implies

some of the info below is definitely helpful for using definitions, but the first bits seem more like the belong in the fetching section